packages/postgres-database/src/simcore_postgres_database/models/resource_tracker_pricing_plan_to_service.py

```diff
@@ -35,13 +35,14 @@
         nullable=False,
         doc="MAJOR.MINOR.PATCH semantic versioning (see https://semver.org)",
     ),
+    column_created_datetime(timezone=True),
+    column_modified_datetime(timezone=True),
     sa.Column(
-        "product",
-        sa.String,
+        "service_default_plan",
```

Comment:
Which layer of the software guarantees that there is only one default per plan?

Reply:
NOTE1: the prevention will be when i create an API for the creation of pricing_plans/units there it should be checked during the creation
NOTE2: currently there are still ongoing discussions on whether there will be 1-2-1 relationship between pricing plans and services (if yes we can even prohibit it in the database, in other words, there will be not needed for the default column)